.vscode/settings.json

```diff
@@ -11,5 +11,6 @@
     "search.exclude": {
         "out": true // set this to false to include "out" folder in search results
     },
-    "typescript.tsdk": "./node_modules/typescript/lib" // we want to use the TS server from our node_modules folder to control its version
+    "typescript.tsdk": "./node_modules/typescript/lib",
+    "python.pythonPath": "C:\\opt\\python27amd64\\python.exe" // we want to use the TS server from our node_modules folder to control its version
```

Comment:
Perhaps we don't need to hardcode the path if we run all the commands under ROS command shell. @kejxu probably knows how to tweak this.

Reply:
that's a great point! the ROS command shell is just a command prompt created with the `env` option set as `extension.env`, as long as a `child_process` is created (via `spawn` or `execute`) with `env: extension.env`, they would have all the ROS environment sourced

meantime, all the changes in `.vscode/settings.json` only affects the local workspace (this means this `settings.json` only takes effect when the parent folder of `.vscode`, which is `vscode-ros`, is opened in a VS Code instance, this is not ROS-related). Maybe `python.pythonPath` does not need to be specified to the one used by ROS when working with the `vscode-ros` project, this file could probably be reverted.